frontend/public/locales/en/renew-child.json

```
@@ -142,7 +142,7 @@
         "save-btn": "Save"
       },
       "error-message": {
-        "federal-provincial-territorial-benefit-required": "Select whether you have federal provincial or territorial dental benefits"
+        "federal-provincial-territorial-benefit-required": "Select whether you have federal, provincial or territorial dental benefits"
```

Since this error message relates to the child's benefits:
```suggestion
        "federal-provincial-territorial-benefit-required": "Select whether the child has federal, provincial or territorial dental benefits"
```